### ISSUE TITLE
tintColor and barTintColor seem to have no effect

### ISSUE THREAD

**Issue by ismyhc** [dfmuir/KINWebBrowser#18]
Hi,

Im trying to change the back text color as well as the back button arrow color. I've trying the following, but i has no effect.

```
let webBrowser = KINWebBrowserViewController()
webBrowser.tintColor = UIColor.redColor()
webBrowser.barTintColor = UIColor.redColor()
webBrowser.showsPageTitleInNavigationBar = false
webBrowser.loadURL(url)
self.navigationController?.pushViewController(webBrowser, animated: true)
```

**Comment by sadon92** [comment]
I had the same problem but with the toolbar icons, had to open the KINWebBrowser .m extension file and poke around until it worked.

**Comment by dfmuir** [comment]
This is resolved in KINWebBrowser 1.2.2
